addon/components/radio-button.js

```diff
@@ -2,7 +2,7 @@ import Ember from 'ember';
 
 const { computed } = Ember;
 
-export default Ember.Component.extend({
+let RadioButton = Ember.Component.extend({
```

Comment:
This should be `const`. For `const`, the variable object can't change, but properties of the variable can still change.